[WIP] Chatspace フロントエンド実装

Daisekkei opened the pull request.

Daisekkei committed:
Fist commitment for Chatspace front implementation

Daisekkei commented:
#What
Chatspaceのフロントを実装する。
Haml、SASSファイルを用いて見本のChatspaceのWebブラウザ見た目と同等の実装をした。

#Why
Chatアプリケーションにおいて、フロントエンドは不可欠な要素であるため

TOMARUMARU commented on app/assets/stylesheets/_chat-main.scss:
```diff
@@ -0,0 +1,109 @@
+    display: flex;
+    justify-content: space-between;
+    
+    .group-info__box {
```
&記法を使いましょう。他の箇所も同様です

Daisekkei replied:
その他のSCSSファイル含めて＆記法へ修正いたしました。

TOMARUMARU commented on app/views/layouts/application.html.haml:
```diff
@@ -0,0 +1,10 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
```
ハッシュロケットではなくシンボル型を使いましょう

Daisekkei replied:
この箇所含めて、
その他のHAMLのファイルにもハッシュロケットを使っておりましたので、修正しました

TOMARUMARU commented on app/views/layouts/mailer.html.haml:
```diff
@@ -0,0 +1,8 @@
+    :css
+      /* Email styles need to be inline */
+  %body
+    = yield
```
mailerファイルは使用しないので消して大丈夫です

Daisekkei replied:
削除いたしました

TOMARUMARU commented on app/views/layouts/mailer.text.haml:
```diff
@@ -0,0 +1 @@
+= yield
```
mailerファイルは使用しないので消して大丈夫です

Daisekkei replied:
削除いたしました

TOMARUMARU commented on config/routes.rb:
```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  get 'messages/index'
```
rootがあるのでこちらは不要です

Daisekkei replied:
get の行を削除いたしました

Daisekkei committed:
1回目レビュー指摘事項の修正

#What
1、SASSを＆記法へ修正
2、Hamlでハッシュロケットをシンボル型へ変更
3、Viewディレクトリのmailerファイルを削除
４、ルーティングファイルのgetの行を削除

#Why
1、コードの量を削減のため
2、ハッシュロケットの記述は古いため
3、今回のアプリケーションにおいて、mailer機能は使用しないため
4、ルーティングにおいて、すでにrootの行で同一機能を表記しており、誤作動を回避するため

shingoteshima commented:
LGTMです！！
